.npmignore

```diff
@@ -0,0 +1,10 @@
+**/*
+!src
```

Comment:
Instead of this file, simply add this to `package.json`:

```
  "files": [
    "src/**",
    "index.js",
    ...
  ],
```

it's more straightforward and simpler to support.